src/venv_manager.rs

```diff
@@ -72,8 +72,11 @@ impl VenvManager {
 
     pub fn install(&self, install_options: &InstallOptions) -> Result<(), Error> {
         print_info_1("Preparing project for developement");
-        if !self.paths.lock.exists() {
-            return Err(Error::MissingLock {});
+        let lock_path = &self.paths.lock;
+        if !lock_path.exists() {
+            return Err(Error::MissingLock {
+                expected_path: lock_path.to_path_buf(),
```

Comment:
Nit: `to_owned` or `into` could also apply here, if you're interested in saving a whopping 7 characters!